Assets/ScriptTemplates/01-C# Templates_OscarSimpleClass-DefaultOscarClass.cs.txt

```diff
@@ -4,22 +4,15 @@
 /// Script used to be copied and add my most common regions, those can be deleted after if unused
 /// </summary>
 /// 
-public class NewScriptStartup
+public class #SCRIPTNAME#
```

Comment:
Remove the script completely, if you added the script template you don't need this anymore.

Reply:
I decided to leave it inside the proyect because I don't want it for ever and if I leave it in the folder with that name is also recognised by unity as a script template (only by this proyect).